app/views/backend/course.erb

```diff
@@ -15,6 +15,7 @@
                 <% @course.chapters.each do |chapter| %>
                    <li>
                     <%= chapter.name %>
+                    <a href="/chapter/delete/<%= chapter.id %>" <i class="material-icons">delete</i></a><br>
```

Comment:
Unneeded break line element!
The chapter.name && chapter.description should be in separated li tags.